### PR TITLE
feat: added support for relational storage

### PR DESCRIPTION
**Pull request by deathsurgeon1**
PR for #8

### CODE REVIEW

**Comment by deathsurgeon1** [comment]
Hi @pushp1997 , please review

**Comment by deathsurgeon1** [comment]
Hi @pushp1997 , can you review this PR once?

**Comment by pushp1997** [comment]
Hye @deathsurgeon1 instead of commenting out mongodb connection can you ask for the user preference? Also, add the RDBMS support in the script as well and not just the notebook?

**Comment by deathsurgeon1** [comment]
Hi @pushp1997 , I have applied the review comments.
please have a look!!

**Review by pushp1997 — APPROVED** [review]

**Comment by pushp1997** [comment]
Thanks @deathsurgeon1, I will merge your PR once the tests go through!

**Comment by deathsurgeon1** [comment]
Hi @pushp1997 , test cases were failing.I have added pysql dependency. Let me know if this will fix the failure.

File: Hackernews Scraper.ipynb [hunk 1]
```diff
@@ -26,7 +26,7 @@
    "source": [
     "import requests\n",
     "from bs4 import BeautifulSoup\n",
-    "from pymongo import MongoClient\n",
+    "#from pymongo import MongoClient\n",
```
Comment (xiaodadalang) [inline Hackernews Scraper.ipynb:29]: Is this a typo?

**Review by pushp1997 — APPROVED** [review]

**Pull request merged**